### ISSUE TITLE
Provide more demos based on different scenarios

### ISSUE THREAD

**Issue by hsluoyz** [casbin/casbin-website#2]
The model syntax is difficult.

We need to provide different scenarios in our docs. Then provide a demo for each scenario with detailed explanation.

**Comment by Kininaru** [comment]
Working on this.

**Comment by pratik2315** [comment]
Hey there ! I see that the above issue requires us to have some development-level experience in casbin to create demos and simplify the model syntax.

**Comment by pratik2315** [comment]
I am a technical writer but I also have a development background in CS. I can work on this issue and write demo's for some scenarios provided some guidance. What'd you think about it @hsluoyz 😄

**Comment by hsluoyz** [comment]
@pratik2315 yes, it requires some development knowledge. You can also get some scenarios from Internet articles and blogs.
